integreat_cms/textlab_api/textlab_api_client.py

```diff
@@ -77,10 +79,23 @@ def benchmark(
         path = f"/benchmark/{text_type}"
         response = self.post_request(path, data, self.token)
 
-        feedback_details = [
-            {"category": cms_name, "result": response.get(textlab_name, [])}
-            for textlab_name, cms_name in textlab_api_feedback_categories.items()
-        ]
+        feedback_details = []
+
+        for textlab_name, cms_name in textlab_api_feedback_categories.items():
```

Comment:
Optional idea from readability perspective:
Maybe it would be better to create a separate mapping for data-terms categories? (with the only one category so far)
And iterate over them sequentially?
I have a feeling that in that case it would be easier to follow/read the code.

Reply:
I tried to make something up and came up with this:

```
        textlab_api_feedback_dataterms = {
            "1289": "abbreviations",
        }

        feedback_details = [
            {"category": cms_name, "result": response.get(textlab_name, [])}
            for textlab_name, cms_name in textlab_api_feedback_categories.items()
        ]
        
        data_terms = response.get('dataTerms', {})

        for dataterm_id, cms_name in textlab_api_feedback_dataterms.items():
            results = data_terms.get(dataterm_id, {}).get('result', [])
            total_length = sum(len(item.get('length', '')) for item in results)
            feedback_details.append({
                "category": cms_name,
                "result": total_length,
        })
```

Not sure if it's any better though.
So just ignore this comment if you want.